src/AccessibilityInsights/Modes/DialogContainerModeControl.xaml.cs

```diff
@@ -57,5 +57,15 @@ public async Task<bool> ShowDialog(ContainedDialog containedDialog)
 
             return await containedDialog.ShowDialog(HideControl).ConfigureAwait(false);
         }
+
+        public void Show(ContainedDialog containedDialog)
```

Comment:
intentional to provide a imperative show/hide experience rather than a blocking showdialog where the dialog is in charge of executing the operation